tags/guide/shadow.ytag

```diff
@@ -0,0 +1,23 @@
+type: embed
+
+colour: black
+embed:
+    title: Shadowing Dependencies (or, Non-Mod Dependencies and You)
+
+    fields:
+        - name: Wait, huh?
+          inline: false
+          value: A recommended way to include a dependency that is **not** a mod (I.E a generic Java library) is to _shadow_ it.
+
+        - name: ...and what does that mean?
+          inline: false
+          value: _Shadowing_ simply means taking a library and including it in your distribution of the thing using said library (in this case, your mod).
+
+        - name: ...and how do I do that?
+          inline: false
+          value: Simple, you read the docs! https://imperceptiblethoughts.com/shadow/introduction/
```

Comment:
I believe most people asking about shadow would like to shadow stuff into their mods. Ideally a solution to that would be nice but is not immediately urgent.

Reply:
Added a short little "crash course" of sorts about `shadow`ing.